lib/common/utils.ts

```diff
@@ -569,3 +569,15 @@ export function findEventTask(target: any, evtName: string): Task[] {
   }
   return result;
 }
+
+Zone[zoneSymbol('patchEventTargetMethods')] = patchEventTargetMethods;
```

Comment:
Yes,I feel the same,the reason to expose those API in such way is

- for easier patch other non standard or 3rd party API
- don't need to import those methods from util.ts